src/systemtest/java/randoop/main/RandoopSystemTest.java

```diff
@@ -113,6 +116,9 @@ public void runCollectionsTest() {
             options,
             timeout);
 
+    long endTime = System.currentTimeMillis();
+    System.out.println("runCollectionTest took: " + (endTime - startTime) + " milliseconds");
```

Comment:
Would you suggest that I comment out the System.currentTimeMillis as well if it isn't being used after I comment out the println?